api-gradle/src/main/kotlin/phraseapp/tasks/deprecated/CheckTask.kt

```diff
@@ -9,6 +9,7 @@ import org.gradle.api.tasks.TaskAction
 import phraseapp.internal.platforms.Platform
 import phraseapp.repositories.checks.CheckRepository
 
+@Deprecated("This task is deprecated, use CheckTask plugin task instead")
```

Comment:
"use tasks.CheckTask task instead"

Add the package because the name of the new one and the old one is the same and remove the plugin keyword, make no sense here.